docs/decisions/release-process.md

```diff
@@ -0,0 +1,67 @@
+# Release Process
+
+## Context and Problem Statement
+
+For most of the Open Food Facts projects we operate with a Staging and Production environment. We typically deploy to Staging every time a change is merged to the default (typically `main`) branch, whereas the Production deployment only happens when we create a Release.
+
+A number of project use the [release-please](https://github.com/googleapis/release-please) action to manage this. This action creates a Release PR which is appended to as each other PR is merged into `main`. Merging the Release PR then triggers a release.
+
+This process is convenient but it does have a few drawbacks:
+* It is not possible to use a `GITHUB_TOKEN` on the release-please action because GitHub then prevents the subsequent deployment action from running when the Release PR is merged. It is therefore necessary to use a personal access token on this action
+* The act of merging the Release PR creates a further change to the code, triggering another build, and it is this new build that is deployed to Production. Hence, what is deployed to Production isn't strictly the same as what was tested in Staging. This could cause issues if third party package dependencies have changed.
```

Comment:
It might be possible to avoid that.

Currently we rebuild the docker images for main and for the tag. And it is the version of the tag that goes to production.

It might be possible to change de build image CI to instead add the tag to the last docker image (corresponding to previous commit) in the case of a v.*.* tag.